integrations/coinbase/src/main/java/org/dash/wallet/integration/coinbase_integration/ui/CoinbaseBuyDashFragment.kt

```diff
@@ -144,26 +145,25 @@ class CoinbaseBuyDashFragment : Fragment(R.layout.fragment_coinbase_buy_dash) {
         }
 
         monitorNetworkChanges()
-        viewModel.isDeviceConnectedToInternet.observe(viewLifecycleOwner){ hasInternet ->
+        viewModel.isDeviceConnectedToInternet.observe(viewLifecycleOwner) { hasInternet ->
             fragment.handleNetworkState(hasInternet)
         }
     }
 
     private fun setupPaymentMethodPayment() {
-        viewModel.activePaymentMethods.observe(viewLifecycleOwner){
-            binding.paymentMethodPicker.paymentMethods = it
-        }
-
-        arguments?.let {
-            viewModel.getActivePaymentMethods(CoinbaseBuyDashFragmentArgs.fromBundle(it).paymentMethods)
+        viewModel.activePaymentMethods.observe(viewLifecycleOwner) {
+            if (it.isNotEmpty()) {
```

Comment:
We should not get to the point of `activePaymentMethods` being empty due to changes in `CoinbaseServicesFragment`, so this is just in case.